src/notaryUtils.js

```diff
@@ -26,11 +26,15 @@ function createCaseID () {
  */
 function setupCase (params) {
   return co(function * () {
-    const uniqueID = params.caseID || uuid()
+    // the caseID param will be in the format of notary/cases/uuid,
+    // only check if its uuid is less than 40 characters
+    const uniqueID = params.caseID ? params.caseID.substring(params.caseID.indexOf('/cases/') + 7) : uuid()
```

Comment:
If the id provided doesn't match the `Uuid` schema the notary will return an error when you try to set up the case (it's checked [here](https://github.com/interledger/five-bells-notary/blob/master/src/controllers/cases.js#L22)). I think the sender should either 1) not check any of the parameters and just return any errors returned by the components or 2) check all of the parameters it's given